scripts/test-internal.js

```diff
@@ -72,7 +72,13 @@ function runTest(name: string, code: string): boolean {
       serialize: true,
       speculate: !modelCode,
       sourceMaps: !!sourceMap,
-    });
+    };
+    if (name.endsWith("/bundle.js~"))
```

Comment:
It uses a naming convention rather than a runtime option. Not something I want to keep, but fixing the runner to deal with options has not yet made it to the top of my to do list.